### PR TITLE
Adds wasps

### PR DESCRIPTION
**Pull request by ChesterTheCheesy**
# General Documentation
### Intent of your Pull Request
I made a simple wasp for the possible future "Jungleland". They are currently only gold slime spawnable.
Wasp sprite was made by Theremin.

### Why is this change good for the game?
More variety = good; puts bees to shame, and most importantly:
Improves RP.

# Wiki Documentation
not sure if it's worth really mentioning unless the mob sees real use in future

### Briefly describe your PR and the impacts of it, in layman's terms. 
This flying simplemob has 200 health, deals 12 damage with 5u ~toxin~ venom injection.
It has a special attack that allows it to charge at the target, easily indicated by the floating "!" symbol.

### What should players be aware of when it comes to the changes your PR is implementing?
There's a new mob in the gold slime pool, i guess.

### What general grouping does this PR fall under? 
possible future mining mob

# Changelog
:cl:  
rscadd: adds wasps  
imageadd: adds wasp sprites
/:cl:

Here's a little preview how they work:

https://user-images.githubusercontent.com/71487903/109632661-95bcb980-7b47-11eb-969e-acfc5904fcb9.mp4

### CODE REVIEW

**Comment by athena413** [comment]
i really like that attack pattern

File: code/modules/mob/living/simple_animal/hostile/mining_mobs/wasp.dm [hunk 1]
```diff
@@ -0,0 +1,85 @@
+//A wasp that charges at their enemy, injecting them with poison
+/mob/living/simple_animal/hostile/asteroid/wasp
+	name = "wasp"
+	desc = "A massive, mutated wasp equipped with a giant stinger. It's eyes flash with burning fury."
```
Comment (UselessTheremin) [inline code/modules/mob/living/simple_animal/hostile/mining_mobs/wasp.dm]: *its, fix your grammar nerd

**Comment by UselessTheremin** [comment]
also make it do the horrible moth scream whenever it attacks the bite sound is boring

**Comment by ChesterTheCheesy** [comment]
Done and done.

**Comment by Marmio64** [comment]
If theyre going to be encountered in swarms, 200 health is too much tbh

**Comment by ChesterTheCheesy** [comment]
200 health is nothing for a mining mob, it's the same as a watcher. If they are going to be spawned in swarms by something, they can have reduced health then. But as a singular mob, i don't think it's that high.

**Comment by Marmio64** [comment]
> 200 health is nothing for a mining mob, it's the same as a watcher. If they are going to be spawned in swarms by something, they can have reduced health then. But as a singular mob, i don't think it's that high.

never said it was high for a singular mob, you know

**Review by Partheo — APPROVED** [review]

**Comment by JamieD1** [comment]
That sprite is horrible
